tests/e2e/utils/src/flows/constants.js

```diff
@@ -2,32 +2,60 @@
  * External dependencies
  */
 const config = require( 'config' );
-
 const baseUrl = config.get( 'url' );
 
+/**
+ * WordPress core dashboard pages.
+ * @type {string}
+ */
 export const WP_ADMIN_LOGIN = baseUrl + 'wp-login.php';
-export const WP_ADMIN_DASHBOARD = baseUrl + 'wp-admin';
-export const WP_ADMIN_PLUGINS = baseUrl + 'wp-admin/plugins.php';
-export const WP_ADMIN_SETUP_WIZARD = baseUrl + 'wp-admin/admin.php?page=wc-admin';
-export const WP_ADMIN_ALL_ORDERS_VIEW = baseUrl + 'wp-admin/edit.php?post_type=shop_order';
-export const WP_ADMIN_ALL_PRODUCTS_VIEW = baseUrl + 'wp-admin/edit.php?post_type=product';
-export const WP_ADMIN_NEW_COUPON = baseUrl + 'wp-admin/post-new.php?post_type=shop_coupon';
-export const WP_ADMIN_NEW_ORDER = baseUrl + 'wp-admin/post-new.php?post_type=shop_order';
-export const WP_ADMIN_NEW_PRODUCT = baseUrl + 'wp-admin/post-new.php?post_type=product';
-export const WP_ADMIN_WC_SETTINGS = baseUrl + 'wp-admin/admin.php?page=wc-settings&tab=';
-export const WP_ADMIN_PERMALINK_SETTINGS = baseUrl + 'wp-admin/options-permalink.php';
-export const WP_ADMIN_NEW_SHIPPING_ZONE = baseUrl + 'wp-admin/admin.php?page=wc-settings&tab=shipping&zone_id=new';
-export const WP_ADMIN_ANALYTICS_PAGES = baseUrl + 'wp-admin/admin.php?page=wc-admin&path=%2Fanalytics%2F';
-export const WP_ADMIN_ALL_USERS_VIEW = baseUrl + 'wp-admin/users.php';
-export const WP_ADMIN_IMPORT_PRODUCTS = baseUrl + 'wp-admin/edit.php?post_type=product&page=product_importer';
-
+export const WP_ADMIN_DASHBOARD = baseUrl + 'wp-admin/';
+export const WP_ADMIN_PLUGINS = WP_ADMIN_DASHBOARD + 'plugins.php';
+export const WP_ADMIN_PERMALINK_SETTINGS = WP_ADMIN_DASHBOARD + 'options-permalink.php';
+export const WP_ADMIN_ALL_USERS_VIEW = WP_ADMIN_DASHBOARD + 'users.php';
+/**
+ * WooCommerce core post type pages.
```

Comment:
I like that you broke these out into logical sections--thanks for making these change!